libbeat/docs/newbeat.asciidoc

```diff
@@ -19,21 +20,32 @@ language, and very few people are experts in it. In fact, several
 people learned Go by contributing to Packetbeat and libbeat, including the
 original Packetbeat authors.
 
+For general information about contributing to Beats, see our
+https://github.com/elastic/beats/blob/master/CONTRIBUTING.md[Contributing Guide].
+
 After you have https://golang.org/doc/install[installed Go] and set up the
 https://golang.org/doc/code.html#GOPATH[GOPATH] environment variable to point to
 your preferred workspace location, a simple way of getting the source code for 
-Beats, including libbeat, and compiling them at the same time is to do:
+Beats, including libbeat and the Beat generator, is to do:
 
 [source,shell]
 ----------------------------------------------------------------------
 go get github.com/elastic/beats
 ----------------------------------------------------------------------
 
+When you run the command, all source files will be downloaded to the
+`$GOPATH/src` path. You can ignore the "no buildable Go source files" message because
+you will build the source later.
+
+//QUESTION: Wondering if adding sentence about "no buildable Go source files" is the best way to handle this. The problem is that users might be confused because the command returns the message "package github.com/elastic/beats: no buildable Go source files in /Users/dedemorton/workspace/src/github.com/elastic/beats". 
```

Comment:
I think it would be nice if we could avoid advising the user to ignore the error. @tsg what do you think?